web/src/components/auth/NeedsAccountModal.tsx

```diff
@@ -30,6 +30,7 @@ export const NeedsAccountModal = (): ReactElement => {
     if (isAuthenticated === IsAuthenticated.TRUE) {
       setShowNeedsAccountModal(false);
     }
+    analytics.event.gen_guidance_stepper_save_modal_displayed.appears.general_guidance_save_modal_displayed();
```

Comment:
[boulder] The `NeedsAccountModal` is used in multiple places throughout the application. Making a change to this component directly means that any time it appears this analytics event will fire. Based on what I'm seeing in the AC, I don't think that's the intention. See if there's another way to get this to fire.